cs/json.texy

```diff
@@ -53,10 +51,14 @@ try {
 }
 \--
 
-Jak odeslat JSON z presenteru?
-==============================
 
-V metodě `action*()` nebo `render*()` zavoláme `$this->sendJson($payload);`, kde `$payload` obsahuje data, která chceme zakódovat do JSONu a odeslat.
+MVC - Presenter
+===============
+
+Jak odeslat JSON z presenteru? V `Presenter::action<>()` metodě zavoláme `$this->sendJson($payload)`, kde `$payload` jsou data, které chceme odeslat v JSON formátu.
```

Comment:
To chápu, ale neměli bychom lhát. Lepší je napsat třeba `
Jak odeslat JSON z presenteru? Lze k tomu použít metodu `$this->sendJson($data)`, kterou můžeme zavolat třeba v `action**()` metodě.